### ISSUE TITLE
Coliforms: New turbidity test procedure

### ISSUE THREAD

**Issue by Rajshreeapatil** [foundation-for-environmental-monitoring/ffem-app#39]
Add new procedure for the coliform test to run multiple tests in parallel
1. Determine test code from QR code from sticker on bottle cap
1. Take photo of the sample at start
1. Repeat above steps after some time
1. Show result which is presence/absence of coliforms
1. Repeat above steps again later if necessary

**Comment by nhternup** [comment]
Project on hold
